### DISCUSSION TITLE
常规设置里的github地址修改.打开后实际并没有

### DISCUSSION THREAD

**Discussion by a393310872** [vernesong/OpenClash#2635] [Q&A]
![image](https://user-images.githubusercontent.com/60805721/185410024-aa909cee-12a8-45ee-9da1-cac7e9c4d253.png)
![image](https://user-images.githubusercontent.com/60805721/185410146-d675b89a-8c37-47ca-9084-c25b119ae92c.png)
这是使用了油猴脚本的效果
![image](https://user-images.githubusercontent.com/60805721/185410235-8e1811f5-a0b8-411a-be59-bf03a20293f4.png)
是绑定网络接口必须启用吗?
![image](https://user-images.githubusercontent.com/60805721/185410439-39b48dc7-7387-479e-a45a-00e515571353.png)

**Comment by vernesong** [comment]
这个是修改插件内置的资源下载地址

**Comment by a393310872** [comment]
好的....感谢了....我以为是替换网页中的链接....倒是方便很多....感谢大佬回复





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vernesong/OpenClash"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月30日(星期二) 下午5:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [vernesong/OpenClash] 常规设置里的github地址修改.打开后实际并没有 (Discussion #2635)





 
这个是修改插件内置的资源下载地址
